pkg/cose/cose.go

```diff
@@ -0,0 +1,183 @@
+package cose
+
+import (
+	"crypto/ecdsa"
+	"crypto/sha256"
+	"crypto/sha512"
+	"crypto/x509"
+	"math/big"
+
+	"github.com/blocky/nitrite/pkg/attestation"
+	"github.com/blocky/nitrite/pkg/nitrite_error"
+	"github.com/fxamacker/cbor/v2"
+)
+
+type CoseHeader struct {
```

Comment:
`COSE` is an acronym https://www.rfc-editor.org/rfc/rfc8152.html. Rename uses of `Cose` to `COSE`